examples/shm.rs

```diff
@@ -0,0 +1,33 @@
+#![allow(clippy::print_literal)]
+
+extern crate procfs;
+
+/// List processes using posix shared memory segments
+
+fn main() {
+    let shared_memory_vec = procfs::Shm::new().unwrap();
+
+    for shared_memory in &shared_memory_vec {
+        println!("key: {}, shmid: {}", shared_memory.key, shared_memory.shmid);
+        println!("============");
+
+        for prc in procfs::process::all_processes().unwrap() {
+            match prc.smaps() {
+                Ok(memory_maps) => {
+                    for (memory_map, memory_map_data) in &memory_maps {
+                        match memory_map.pathname {
+                            procfs::process::MMapPath::Vsys(key) => {
```

Comment:
enum variant `MMapPath::Vsys` not found?

Reply:
MMapPath::Vsys compile error(not found)

```
origin  https://github.com/tatref/procfs.git (fetch)
origin  https://github.com/tatref/procfs.git (push)
commit a0679242e613e4618cb92a0ed70721af218efa7d (HEAD -> sysvipc_shm, origin/sysvipc_shm)
Author: tatref <tatref@users.noreply.github.com>
Date:   Sat Nov 6 19:13:11 2021 +0100

    Update shm.rs
```

Reply:
I think you're missing  #151

Reply:
I rebased this PR from current master